src/test/java/de/mediathekview/mserver/base/utils/UrlUtilsTestGetBaseUrl.java

```diff
@@ -9,7 +9,7 @@
 import org.junit.runners.Parameterized;
 
 @RunWith(Parameterized.class)
-public class UrlUtilsTest {
+public class UrlUtilsTestGetBaseUrl {
   @Parameterized.Parameters
   public static Collection<Object[]> data() {
     return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)